easybuild/easyconfigs/g/GetFEM/GetFEM-5.4.1-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,59 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'ConfigureMake'
+
+name = 'GetFEM'
+version = '5.4.1'
+local_pyver = '3.7.4'
+versionsuffix = '-Python-%(local_pyver)s'
+
+homepage = 'http://getfem.org'
+description = """GetFEM is an open source library based on collaborative development. It aims to
+offer the most flexible framework for solving potentially coupled systems of linear and nonlinear
+partial differential equations with the finite element method"""
+
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+builddependencies = [
+    ('Autoconf', '2.69', '', ('GCCcore', '8.3.0')),
+    ('Automake', '1.16.1', '', ('GCCcore', '8.3.0')),
+    ('Autotools', '20180311', '', ('GCCcore', '8.3.0')),
+]
+
+dependencies = [
+    ('Qhull', '2019.1', '', ('GCCcore', '8.3.0')),
+    ('MUMPS', '5.2.1', '', ('foss', '2019b-metis')),
+    ('SciPy-bundle', '2019.10', '', ('foss', '2019b-Python-%(local_pyver)s')),
```

Comment:
With the changes listed above, please change this to be `versionsuffix`, in order to pick up the correct `SciPy-bundle` dependency.